pkg/frr/configure.go

```diff
@@ -91,15 +93,26 @@ func (f *FRRManager) renderSubtemplates(in FRRConfiguration) (*FRRTemplateConfig
 	if err != nil {
 		return nil, err
 	}
+	envECMPMaximumPaths, isSet := os.LookupEnv("NWOP_ECMP_MAXIMUM_PATHS")
```

Comment:
FYI: The second return value is commonly called `ok` for things like this, but no need to change that.